src/priv-csrs.adoc

```diff
@@ -282,8 +282,26 @@ Supervisor count overflow.
 
 |`0x5A8` |SRW |`scontext` |Supervisor-mode context register.
 
-4+^|Supervisor Resource Management Configuration
-|`0x181` |SRW |`srmcfg` |Supervisor Resource Management Configuration.
+//4+^|Supervisor Resource Management Configuration
+//|`0x181` |SRW |`srmcfg` |Supervisor Resource Management Configuration.
```

Comment:
@ved-rivos Any reason why this is commented out? Should I just make a PR to undo this change?

Reply:
https://github.com/riscv/riscv-isa-manual/issues/2027

Reply:
@aswaterman - it was commented out waiting for Priv 1.13 to be released. It can be uncommented now.

Reply:
Thanks, I'll handle it.